scripts/jobs/cron_tasks.inc.http.10.apache.php

```diff
@@ -52,6 +52,25 @@ public function reload() {
 		}
 		$this->logger->logAction(CRON_ACTION, LOG_INFO, 'apache::reload: reloading apache');
 		safe_exec(escapeshellcmd(Settings::Get('system.apachereload_command')));
+
+		// Config test
+		$return = -999;
+		$cmds = [ 'apache2ctl', 'apache2', 'httpd2', 'httpd', 'apache' ];
+		foreach ($cmds as $cmd) {
+			$out = safe_exec($cmd . ' -t 2>&1 ', $return, [ '&','>' ]);
+			if ($return == 127) {
+				continue;
+			} else if ($return > 0 && is_array($out) && count($out) > 0) {
+				// Always log this error, regardless of the settings
+				$log = Settings::Get('logger.log_cron');
```

Comment:
remembering the value of log_cron and setting it to 1 to log the action should not be necessary as warnings, errors, and critical messages **will** be logged anyway, see https://github.com/Froxlor/Froxlor/blob/master/lib/classes/logger/class.FroxlorLogger.php#L114